source/includes/_sequences.md

```diff
@@ -122,20 +141,352 @@ padding      | body    | Max number of digits the code entries will have (they w
+if err != nil {
+    // Deal with possible network errors here...
+}
+
+var cs CodeCollection
+decodingErr := json.NewDecoder(res.Body).decode(&cs)
+```
+
+> Response
+
+```json
+{
+    "codes": [
+        {
+            "id": "d4cfb49f-035b-46f0-b7b4-ec733aee2d00",
+            "name": "Some series",
+            "prefix": "XX",
+            "padding": 7,
+            "suffix": "Y"
+        },
+        ...
+    ]
+}
+```
+
+Retrieves all the code objects belonging to the owner.
+
+### HTTP Request
+
+`GET /sequence/:owner_id/codes`
+
+### Parameters
+
+Parameter    | Type    | Description
+------------ | ------- | -----------
+owner_id     | path    | UUID of the company that created the codes
 
 ## Create entry for a code
 
-Something here
+```shell
+curl --request POST "https://api.invopop.dev/sequence/:owner_id/code/:code_id/entry" \
+  -H "Authorization: Bearer your.api.key"
+  -H "Content-Type: application/json"
+  -d '{
+        "id": "175ec7e5-8c14-46fd-90ff-ccb37d71550c",
+        "meta": {
+          "key": "value"
+        }
+      }'
+```
+
+```go
+import (
+    "bytes"
+    "encoding/json"
+    "net/http"
+)
+
+type Entry struct {
+	ID     string            `json:"id"`
+	CodeID string            `json:"code_id"`
+	Idx    int64             `json:"idx"`
+	Value  string            `json:"value"`
+	Meta   map[string]string `json:"meta,omitempty"`
+	PrevID string            `json:"prev_id,omitempty"`
+}
+
+newEntry := Entry{
+    ID: "175ec7e5-8c14-46fd-90ff-ccb37d71550c"
+    Meta: map[string]string{
+        "key": "value",
+    }
+}
+newEntryBytes, encodingError := json.Marshall(newEntry)
+if encodingError != nil {
+    // Deal with possible encoding errors
+}
+
+httpClient := &http.Client{}
+data := bytes.NewBuffer(newEntryBytes)
+req, _ := http.NewRequest("POST", "https://api.invopop.dev/sequence/:owner_id/code/:code_id/entry", data)
+req.Header.Set("Authorization", "Bearer your.api.token")
+req.Header.Set("Content-Type", "application/json")
+
+res, err := client.Do(req)
+if err != nil {
+    // Deal with possible network errors here...
+}
+
+
+var entry Entry
+decodingErr := json.NewDecoder(res.Body).decode(&entry)
+```
+
+> Response
+
+```json
+{
+    "id": "175ec7e5-8c14-46fd-90ff-ccb37d71550c",
+    "code_id": "...",
+    "idx": 1,
+    "value": "XX0000001Y",
+    "meta": {
+        "key": "value"
+    }
+}
+```
+
+
+Creates a new entry object, following the properties and sequence of the specified code.
+
+### HTTP Request
+
+`POST /sequence/:owner_id/code/:code_id/entry`
+
+### Parameters
+
+Parameter    | Type    | Description
+------------ | ------- | -----------
+owner_id     | path    | UUID of the company that created the code
+code_id      | path    | UUID that identifies the code
+id           | body    | UUID that will identify the entry
+meta         | body    | Metadata object
 
 ## Retrieve entry for a code
 
-Something here
+```shell
+curl "https://api.invopop.dev/sequence/:owner_id/code/:code_id/entries" \
```

Comment:
This seems to be returning all the entries of a code and not an specific entry like the title suggests.